### PR TITLE
Add pipeleline clean scripts

### PR DESCRIPTION
**Pull request by poornima-sivanand**

### CODE REVIEW

File: lib/mergePR.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    const username = settings.phases['build'].credentials.idir.user 
+    const password = settings.phases['build'].credentials.idir.pass
+    const url = settings.bitbucketUrl
+    const pr = settings.options.pr
+    const repoName = settings.options.repoName
+    const projectName = settings.options.projectName
+    const p1 = spawn('curl', ['-H', 'Content-type: application/json', '-u', `${username}:${password}`,'-X', 'GET', `${url}/rest/api/latest/projects/${projectName}/repos/${repoName}/pull-requests/${pr}/merge` ])
+    let p2 = spawn('jq', ['.canMerge'], {stdio: [p1.stdout, 'pipe', process.error]})
+    p2.stdout.on( 'data', data => {
+        if(`${data}`){
```
Comment (ianliu28) [inline lib/mergePR.js]: is the intention if(data) or if(`${data}`) ?

File: lib/mergePR.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            const p3 = spawn('curl', ['-H', 'Content-type: application/json', '-u', `${username}:${password}`,'-X', 'POST', `${url}/rest/api/latest/projects/${projectName}/repos/${repoName}/pull-requests/${pr}/merge?version` ])
```
Comment (ianliu28) [inline lib/mergePR.js]: Didn't use git curl api before but should this be 'GET' not 'POST'?

File: lib/mergePR.js (hunk 1 above)
Comment (poornima-sivanand) [reply lib/mergePR.js]: It is ${data}

File: lib/mergePR.js (hunk 2 above)
Comment (poornima-sivanand) [reply lib/mergePR.js]: No, in this scenario, this is how we cant get the merge version using POST

**Pull request merged**